package.json

```diff
@@ -125,7 +128,7 @@
     "@angular/language-service": "~17.2.2",
     "@capacitor/cli": "^5.5.1",
     "@compodoc/compodoc": "^1.1.23",
-    "@ionic/angular-toolkit": "^10.0.0",
+    "@ionic/angular-toolkit": "^11.0.0",
```

Comment:
thanks for catching, we've been on angular 17 for a while which should have required this version. Interesting that there weren't more notifications from the cli to update (or maybe I just didn't notice them) - out of interest, how did you realise?

Reply:
I actually can't remember what led me to change this, but I think an error with the open-layers components and a Google search.

I've included this upgrade in #2374